installation error: Operation failed due to runtime error: Cluster name id is None!

argo installation on existing kubernetes(AWS) run into following error. I uninstall and reinstall the argo-only on the existing cluster many times, not sure if my kubernetes is messed up and caused this problem?! 

```

Welcome to Argo.
Press 'tab' or type 'help' to see a list of options. You can start with 'install'.

If you like us, show us your support #argoproj
argo cluster ops> argocluster  uninstall-argo-only --kubeconfig /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf
2017-10-31T20:28:25 INFO ax.cluster_management.argo_cluster_manager MainThread: Uninstalling Argo platform ...
2017-10-31T20:28:27 INFO ax.cluster_management.argo_cluster_manager MainThread: Done!
argo cluster ops> argocluster  uninstall-argo-only --kubeconfig /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf^C
argo cluster ops> argocluster  install-argo-only --kubeconfig /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf --cloud-provider aws --cluster-name sb3-c2.francis.applatix.net --cloud-profile sb3 --cluster-bucket applatixtestsb3^C-cloud-region us-west-2
argo cluster ops> argocluster  install-argo-only --cluster-name sb3-c2.francis.applatix.net --cloud-provider aws --cloud-profile sb3 --cluster-bucket applatixtestsb3 --cloud-region us-west-2 --kubeconfig /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf
2017-10-31T20:31:16 INFO ax.cluster_management.argo_cluster_manager MainThread: Installing Argo platform ...
2017-10-31T20:31:16 INFO ax.cluster_management.argo_cluster_manager MainThread: s3 bucket endpoint: None
2017-10-31T20:31:18 INFO ax.cluster_management.app.options.install_options MainThread: Cloud placement not provided, setting it to us-west-2b from currently available zones ['us-west-2a', 'us-west-2b', 'us-west-2c']
2017-10-31T20:31:18 INFO ax.meta.cluster_id MainThread: Instantiating cluster bucket ...
2017-10-31T20:31:27 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:31 INFO ax.meta.cluster_id MainThread: Cannot find cluster name/id mapping from bucket
2017-10-31T20:31:31 INFO ax.cluster_management.app.common MainThread: Cannot find cluster name id: Failed to load cluster name and cluster id from bucket. Cluster is not yet created.
2017-10-31T20:31:31 INFO ax.meta.cluster_id MainThread: Cluster id not provided, generate one.
2017-10-31T20:31:31 INFO ax.meta.cluster_id MainThread: Created new name-id sb3-c2.francis.applatix.net-7ad2a9e6-be7a-11e7-b037-025000000001
2017-10-31T20:31:31 INFO ax.meta.config_s3_path MainThread: Using AX cluster config path applatixtestsb3
2017-10-31T20:31:34 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:39 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:40 INFO ax.platform.ax_cluster_info MainThread: Downloading cluster current state ...
2017-10-31T20:31:42 INFO ax.platform.cluster_buckets MainThread: Creating and updating all cluster buckets ...
2017-10-31T20:31:42 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:44 INFO ax.cloud.aws.aws_s3 MainThread: Bucket applatixtestsb3 already exist, don't create.
2017-10-31T20:31:44 INFO ax.platform.cluster_buckets MainThread: Created applatixtestsb3 bucket ... DONE
2017-10-31T20:31:44 INFO ax.meta.config_s3_path MainThread: Using AX cluster data path applatixtestsb3
2017-10-31T20:31:44 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:46 INFO ax.cloud.aws.aws_s3 MainThread: Bucket applatixtestsb3 already exist, don't create.
2017-10-31T20:31:46 INFO ax.platform.cluster_buckets MainThread: Created applatixtestsb3 bucket ... DONE
2017-10-31T20:31:46 INFO ax.platform.cluster_buckets MainThread: Creating and updating all cluster buckets ... DONE
2017-10-31T20:31:46 INFO ax.platform.ax_cluster_info MainThread: Setting kube_config to /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf
2017-10-31T20:31:46 INFO ax.platform.cluster_config.cluster_config MainThread: Uploading cluster config to s3 at sb3-c2.francis.applatix.net/7ad2a9e6-be7a-11e7-b037-025000000001/cluster_config ...
2017-10-31T20:31:48 INFO ax.platform.cluster_config.cluster_config MainThread: Uploading cluster config to s3 ... DONE
2017-10-31T20:31:48 INFO ax.platform.ax_cluster_info MainThread: Saving kubeconfig to s3 ...
2017-10-31T20:31:50 INFO ax.platform.ax_cluster_info MainThread: Saved kubeconfig /tmp/ax_kube/cluster_sb3-c2.francis.applatix.net.conf at applatixtestsb3/sb3-c2.francis.applatix.net/7ad2a9e6-be7a-11e7-b037-025000000001/kube_config
2017-10-31T20:31:50 INFO ax.cluster_management.app.cluster_installer MainThread: Starting platform install
2017-10-31T20:31:50 INFO ax.platform.platform MainThread: Using Kubernetes manifest from /ax/config/service/argo-all
2017-10-31T20:31:50 INFO ax.platform.platform MainThread: Using platform configuration "Argo Platform Config Only" from /ax/config/service/config/argo-all-platform-bootstrap.cfg
2017-10-31T20:31:53 INFO ax.cloud.aws.aws_s3 MainThread: Using region us-west-2 for bucket applatixtestsb3
2017-10-31T20:31:54 INFO ax.platform.ax_monitor_helper kube-events-monitor: Thread kube-events-monitor: Starts
2017-10-31T20:31:54 INFO ax.platform.ax_monitor_helper kube-pods-monitor: Thread kube-pods-monitor: Starts
2017-10-31T20:31:54 INFO ax.platform.ax_monitor_helper kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: Starts
2017-10-31T20:31:54 INFO ax.platform.ax_monitor_helper kube-persistentvolumes-monitor: Thread kube-persistentvolumes-monitor: Starts
2017-10-31T20:31:54 INFO ax.platform.ax_monitor_helper kube-services-monitor: Thread kube-services-monitor: Starts
2017-10-31T20:31:55 INFO ax.platform.cluster_config.cluster_config MainThread: Uploading cluster config to s3 at sb3-c2.francis.applatix.net/7ad2a9e6-be7a-11e7-b037-025000000001/cluster_config ...
2017-10-31T20:31:57 INFO ax.platform.cluster_config.cluster_config MainThread: Uploading cluster config to s3 ... DONE
2017-10-31T20:31:57 DEBUG ax.platform.platform MainThread: Replacing ENVs: {'AX_CLUSTER_INSTALL_VERSION': '1.1.0-dacad13-bf7a0c7', 'SANDBOX_ENABLED': 'None', 'ARGO_S3_ACCESS_KEY_ID': '', 'AX_AWS_ACCOUNT': '139388712480', 'AX_CUSTOMER_ID': 'user-customer-id', 'NEW_KUBE_SALT_SHA1': ' ', 'ARGO_DATA_BUCKET_NAME': 'applatixtestsb3', 'AX_KUBE_VERSION': '1.6.7', 'NAMESPACE': 'axsys', 'LOAD_BALANCER_TYPE': 'LoadBalancer', 'NEW_KUBE_SERVER_SHA1': ' ', 'ARGO_S3_ACCESS_KEY_SECRET': '', 'DNS_SERVER_IP': '192.168.0.10', 'VERSION': '1.1.0-96d4b4c', 'AX_CLUSTER_META_URL_V1': 'https://s3-us-west-2.amazonaws.com/applatixtestsb3/sb3-c2.francis.applatix.net/7ad2a9e6-be7a-11e7-b037-025000000001/metadata/v1/metadata.yaml', 'REGISTRY': 'docker.applatix.net', 'TRUSTED_CIDR': '["0.0.0.0/0"]', 'REGISTRY_SECRETS': 'ewoJImF1dGhzIjogewoJCSJkb2NrZXIuYXBwbGF0aXgubmV0IjogewoJCQkiYXV0aCI6ICJZWGhrWlhZNk5HeDZiek5NY1haamJWUjMiCgkJfQoJfQp9Cg==', 'AX_CLUSTER_NAME_ID': 'sb3-c2.francis.applatix.net-7ad2a9e6-be7a-11e7-b037-025000000001', 'ARGO_LOG_BUCKET_NAME': 'applatixtestsb3', 'AX_AWS_REGION': 'us-west-2'}
2017-10-31T20:31:57 INFO ax.platform.platform MainThread: Create step: Essential Kubernetes Namespaces
2017-10-31T20:31:57 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axuser-namespace          |  /ax/config/service/argo-all/axuser-namespace.yml.in                                                            |  None
axsys-namespace           |  /ax/config/service/argo-all/axsys-namespace.yml.in                                                             |  None

2017-10-31T20:32:02 INFO ax.platform.platform Thread-8: Creating axsys-namespace in namespace None ...
2017-10-31T20:32:02 INFO ax.platform.platform Thread-7: Creating axuser-namespace in namespace None ...
2017-10-31T20:32:13 INFO ax.kubernetes.kube_object Thread-7: namespace "axuser" created
2017-10-31T20:32:13 INFO ax.platform.platform Thread-7: Successfully created axuser-namespace.
2017-10-31T20:32:13 INFO ax.kubernetes.kube_object Thread-8: namespace "axsys" created
2017-10-31T20:32:13 INFO ax.platform.platform Thread-8: Successfully created axsys-namespace.
2017-10-31T20:32:14 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axuser-namespace          |  axuser-namespace:OK                                                                                            |  10.97
axsys-namespace           |  axsys-namespace:OK                                                                                             |  11.35

======= WARNING EVENTS =======
==============================

2017-10-31T20:32:14 INFO ax.platform.platform MainThread: Create step: Basic Infrastructure
2017-10-31T20:32:14 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
redis-pvc                 |  /ax/config/service/argo-all/redis-pvc.yml.in                                                                   |  axsys
axops-svc                 |  /ax/config/service/argo-all/axops-svc.yml.in                                                                   |  axsys
applatix-svc              |  /ax/config/service/argo-all/applatix-svc.yml.in                                                                |  axsys
gateway-pvc               |  /ax/config/service/argo-all/gateway-pvc.yml.in                                                                 |  axsys
kafka-zk-pvc              |  /ax/config/service/argo-all/kafka-zk-pvc.yml.in                                                                |  axsys
axops-internal-svc        |  /ax/config/service/argo-all/axops-internal-svc.yml.in                                                          |  axsys

2017-10-31T20:32:16 INFO ax.platform.platform Thread-14: Creating kafka-zk-pvc in namespace axsys ...
2017-10-31T20:32:17 INFO ax.platform.platform Thread-13: Creating axops-svc in namespace axsys ...
2017-10-31T20:32:18 INFO ax.platform.platform Thread-16: Creating gateway-pvc in namespace axsys ...
2017-10-31T20:32:19 INFO ax.platform.platform Thread-15: Creating applatix-svc in namespace axsys ...
2017-10-31T20:32:19 INFO ax.platform.platform Thread-12: Creating redis-pvc in namespace axsys ...
2017-10-31T20:32:19 INFO ax.platform.platform Thread-17: Creating axops-internal-svc in namespace axsys ...
2017-10-31T20:32:19 INFO ax.kubernetes.kube_object Thread-13: service "axops" created
2017-10-31T20:32:19 INFO ax.platform.ax_monitor kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: object kdata-1:persistentvolumeclaims has reached expected status
2017-10-31T20:32:19 INFO ax.kubernetes.kube_object Thread-14: persistentvolumeclaim "kdata-1" created
2017-10-31T20:32:19 INFO ax.kubernetes.kube_object Thread-14: persistentvolumeclaim "kdata-2" created
2017-10-31T20:32:19 INFO ax.kubernetes.kube_object Thread-14: persistentvolumeclaim "kdata-3" created
2017-10-31T20:32:19 INFO ax.platform.platform Thread-14: Successfully created kdata-1 with code OK.
2017-10-31T20:32:19 INFO ax.platform.ax_monitor kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: object kdata-2:persistentvolumeclaims has reached expected status
2017-10-31T20:32:19 INFO ax.platform.platform Thread-14: Successfully created kdata-2 with code OK.
2017-10-31T20:32:20 INFO ax.platform.platform Thread-13: Successfully created axops-svc.
2017-10-31T20:32:20 INFO ax.kubernetes.kube_object Thread-16: persistentvolumeclaim "gateway-persistent-storage" created
2017-10-31T20:32:20 INFO ax.platform.ax_monitor kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: object kdata-3:persistentvolumeclaims has reached expected status
2017-10-31T20:32:20 INFO ax.platform.platform Thread-14: Successfully created kdata-3 with code OK.
2017-10-31T20:32:20 INFO ax.platform.ax_monitor kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: object gateway-persistent-storage:persistentvolumeclaims has reached expected status
2017-10-31T20:32:20 INFO ax.platform.platform Thread-16: Successfully created gateway-persistent-storage with code OK.
2017-10-31T20:32:21 INFO ax.platform.platform Thread-16: Successfully created object gateway-pvc.
2017-10-31T20:32:21 INFO ax.platform.platform Thread-14: Successfully created object kafka-zk-pvc.
2017-10-31T20:32:22 INFO ax.kubernetes.kube_object Thread-15: service "applatix" created
2017-10-31T20:32:22 INFO ax.platform.platform Thread-15: Successfully created applatix-svc.
2017-10-31T20:32:22 INFO ax.kubernetes.kube_object Thread-17: service "axops-internal" created
2017-10-31T20:32:22 INFO ax.platform.platform Thread-17: Successfully created axops-internal-svc.
2017-10-31T20:32:22 INFO ax.kubernetes.kube_object Thread-12: persistentvolumeclaim "redis-pvc" created
2017-10-31T20:32:23 INFO ax.platform.ax_monitor kube-persistentvolumeclaims-monitor: Thread kube-persistentvolumeclaims-monitor: object redis-pvc:persistentvolumeclaims has reached expected status
2017-10-31T20:32:23 INFO ax.platform.platform Thread-12: Successfully created redis-pvc with code OK.
2017-10-31T20:32:23 INFO ax.platform.platform Thread-12: Successfully created object redis-pvc.
2017-10-31T20:32:23 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
kafka-zk-pvc              |  kdata-1:OK / kdata-2:OK / kdata-3:OK                                                                           |  5.75
applatix-svc              |  applatix-svc:OK                                                                                                |  3.14
axops-internal-svc        |  axops-internal-svc:OK                                                                                          |  3.49
redis-pvc                 |  redis-pvc:OK                                                                                                   |  4.21
axops-svc                 |  axops-svc:OK                                                                                                   |  3.09
gateway-pvc               |  gateway-persistent-storag:OK                                                                                   |  3.39

======= WARNING EVENTS =======
==============================

Error from server (NotFound): configmaps "cluster-dns-name" not found
2017-10-31T20:32:24 ERROR ax.platform.platform MainThread: Failed to get cluster dns name from config map.
2017-10-31T20:32:24 INFO ax.platform.platform MainThread:

>>>>> Starting Argo platform... cluster DNS: a9761159dbe7a11e7ab7a0af8e172eed-1574686931.us-west-2.elb.amazonaws.com

2017-10-31T20:32:24 INFO ax.platform.platform MainThread:

=======================================================
Platform Up: Bringing up Argo services...
Image Namespace: axsys
Image Version: 1.1.0-96d4b4c
=======================================================

2017-10-31T20:32:24 INFO ax.platform.platform MainThread: Create step: Registry Secrets
2017-10-31T20:32:24 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
registry-secrets-axuser   |  /ax/config/service/argo-all/registry-secrets.yml.in                                                            |  axuser
registry-secrets-axsys    |  /ax/config/service/argo-all/registry-secrets.yml.in                                                            |  axsys
registry-secrets-kube     |  /ax/config/service/argo-all/registry-secrets.yml.in                                                            |  kube-system

2017-10-31T20:32:26 INFO ax.platform.platform Thread-26: Creating registry-secrets-axuser in namespace axuser ...
2017-10-31T20:32:26 INFO ax.platform.platform Thread-27: Creating registry-secrets-kube in namespace kube-system ...
2017-10-31T20:32:28 INFO ax.platform.platform Thread-28: Creating registry-secrets-axsys in namespace axsys ...
2017-10-31T20:32:28 INFO ax.kubernetes.kube_object Thread-26: secret "applatix-registry" created
2017-10-31T20:32:29 INFO ax.platform.platform Thread-26: Successfully created registry-secrets-axuser.
2017-10-31T20:32:30 INFO ax.kubernetes.kube_object Thread-28: secret "applatix-registry" created
2017-10-31T20:32:31 INFO ax.platform.platform Thread-28: Successfully created registry-secrets-axsys.
2017-10-31T20:32:31 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
registry-secrets-axsys    |  registry-secrets-axsys:OK                                                                                      |  2.25
registry-secrets-axuser   |  registry-secrets-axuser:OK                                                                                     |  2.36
registry-secrets-kube     |  registry-secrets-kube:OBJ_EXISTS                                                                               |  0.39

======= WARNING EVENTS =======
==============================

2017-10-31T20:32:31 INFO ax.platform.platform MainThread: Create step: Kafka
2017-10-31T20:32:31 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
kafka-zk-svc              |  /ax/config/service/argo-all/kafka-zk-svc.yml.in                                                                |  axsys

2017-10-31T20:32:36 INFO ax.platform.platform Thread-32: Creating kafka-zk-svc in namespace axsys ...
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: service "kafka-zk" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: service "kafka-zk-1" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: service "kafka-zk-2" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: service "kafka-zk-3" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: deployment "kafka-zk-1" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: deployment "kafka-zk-2" created
2017-10-31T20:32:43 INFO ax.kubernetes.kube_object Thread-32: deployment "kafka-zk-3" created
2017-10-31T20:33:03 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object kafka-zk-2:pods has reached expected status
2017-10-31T20:33:03 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object kafka-zk-1:pods has reached expected status
2017-10-31T20:33:03 INFO ax.platform.platform Thread-32: Successfully created kafka-zk-1 with code WARN.
2017-10-31T20:33:03 INFO ax.platform.platform Thread-32: Successfully created kafka-zk-2 with code WARN.
2017-10-31T20:33:05 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object kafka-zk-3:pods has reached expected status
2017-10-31T20:33:05 INFO ax.platform.platform Thread-32: Successfully created kafka-zk-3 with code WARN.
2017-10-31T20:33:07 INFO ax.platform.platform Thread-32: Successfully created object kafka-zk-svc.
2017-10-31T20:33:07 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
kafka-zk-svc              |  kafka-zk-1:WARN / kafka-zk-2:WARN / kafka-zk-3:WARN                                                            |  31.14

======= WARNING EVENTS =======
kafka-zk-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'kafka-zk-1-1334432997-nvdvv'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-97212842-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'kafka-zk-1-1334432997-nvdvv'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'kafka-zk-2-1556152109-fgrd2'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-975e8569-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'kafka-zk-2-1556152109-fgrd2'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'kafka-zk-3-314149381-vsh0z'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-978f51cd-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'kafka-zk-3-314149381-vsh0z'}]

==============================

2017-10-31T20:33:07 INFO ax.platform.platform MainThread: Create step: Cassandra
2017-10-31T20:33:07 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axdb-svc                  |  /ax/config/service/argo-all/axdb-svc.yml.in                                                                    |  axsys

2017-10-31T20:33:11 INFO ax.platform.platform Thread-39: Creating axdb-svc in namespace axsys ...
2017-10-31T20:33:15 INFO ax.kubernetes.kube_object Thread-39: service "axdb" created
2017-10-31T20:33:15 INFO ax.kubernetes.kube_object Thread-39: statefulset "axdb" created
2017-10-31T20:33:36 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axdb-0:pods has reached expected status
2017-10-31T20:33:36 INFO ax.platform.platform Thread-39: Successfully created axdb-0 with code WARN.
2017-10-31T20:34:12 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axdb-1:pods has reached expected status
2017-10-31T20:34:12 INFO ax.platform.platform Thread-39: Successfully created axdb-1 with code WARN.
2017-10-31T20:34:43 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axdb-2:pods has reached expected status
2017-10-31T20:34:43 INFO ax.platform.platform Thread-39: Successfully created axdb-2 with code WARN.
2017-10-31T20:34:43 INFO ax.platform.platform Thread-39: Successfully created object axdb-svc.
2017-10-31T20:34:43 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axdb-svc                  |  axdb-0:WARN / axdb-1:WARN / axdb-2:WARN                                                                        |  92.3

======= WARNING EVENTS =======
axdb-svc: [{'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-0" (repeated 5 times)', 'name': u'axdb-0'}, {'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-0" (repeated 5 times)', 'name': u'axdb-0'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-b8bd934f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0c89a76dff036f3fc" to instance "i-06f583ad10d23c14f": IncorrectState: vol-0c89a76dff036f3fc is not \'available\'.\n\tstatus code: 400, request id: 8d56a54d-b9f0-4747-a944-780069572821', 'name': u'axdb-0'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axdb-0'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-b8bd934f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0c89a76dff036f3fc" to instance "i-06f583ad10d23c14f": IncorrectState: vol-0c89a76dff036f3fc is not \'available\'.\n\tstatus code: 400, request id: e59cd8c9-0915-4867-b7b6-36acb498d845', 'name': u'axdb-0'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-b8bd934f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0c89a76dff036f3fc" to instance "i-06f583ad10d23c14f": IncorrectState: vol-0c89a76dff036f3fc is not \'available\'.\n\tstatus code: 400, request id: 6c75d082-4e9d-45a3-987b-5b2c76f75229', 'name': u'axdb-0'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-b8bd934f-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'axdb-0'}, {'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-1" (repeated 5 times)', 'name': u'axdb-1'}, {'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-1" (repeated 5 times)', 'name': u'axdb-1'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axdb-1'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-ce41f52f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0efd703226ad9dad1" to instance "i-0a933419f07707430": IncorrectState: vol-0efd703226ad9dad1 is not \'available\'.\n\tstatus code: 400, request id: 43f36ac2-c5aa-41fd-abc1-5151c82ae5d1', 'name': u'axdb-1'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-ce41f52f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0efd703226ad9dad1" to instance "i-0a933419f07707430": IncorrectState: vol-0efd703226ad9dad1 is not \'available\'.\n\tstatus code: 400, request id: 4f848de6-a844-484d-ad0a-d4a08b6c0add', 'name': u'axdb-1'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-ce41f52f-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-0efd703226ad9dad1" to instance "i-0a933419f07707430": IncorrectState: vol-0efd703226ad9dad1 is not \'available\'.\n\tstatus code: 400, request id: be520cec-0118-465a-badd-d687f42ae4f8', 'name': u'axdb-1'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-ce41f52f-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'axdb-1'}, {'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-2" (repeated 5 times)', 'name': u'axdb-2'}, {'reason': u'FailedScheduling', 'message': u'PersistentVolumeClaim is not bound: "pdata-axdb-2" (repeated 5 times)', 'name': u'axdb-2'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axdb-2'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-e0d119a8-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-04b8bd699aa1e9918" to instance "i-04eb24ddebaedef61": IncorrectState: vol-04b8bd699aa1e9918 is not \'available\'.\n\tstatus code: 400, request id: 149f3bc3-4ae8-4ad7-8481-31b7e3642ec0', 'name': u'axdb-2'}, {'reason': u'FailedMount', 'message': u'AttachVolume.Attach failed for volume "pvc-e0d119a8-be7a-11e7-ab7a-0af8e172eeda" : Error attaching EBS volume "vol-04b8bd699aa1e9918" to instance "i-04eb24ddebaedef61": IncorrectState: vol-04b8bd699aa1e9918 is not \'available\'.\n\tstatus code: 400, request id: 0d3c1a15-1aef-4b55-b492-02e9ef76860d', 'name': u'axdb-2'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-e0d119a8-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'axdb-2'}]

==============================

2017-10-31T20:34:43 INFO ax.platform.platform MainThread: Create step: Heavy Hitters
2017-10-31T20:34:43 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axops                     |  /ax/config/service/argo-all/axops.yml.in                                                                       |  axsys
gateway-svc               |  /ax/config/service/argo-all/gateway-svc.yml.in                                                                 |  axsys
axmon-svc                 |  /ax/config/service/argo-all/axmon-svc.yml.in                                                                   |  axsys

2017-10-31T20:34:43 INFO ax.platform.platform Thread-46: Creating axops in namespace axsys ...
2017-10-31T20:34:43 INFO ax.platform.platform Thread-48: Creating axmon-svc in namespace axsys ...
2017-10-31T20:34:44 INFO ax.platform.platform Thread-47: Creating gateway-svc in namespace axsys ...
2017-10-31T20:34:46 INFO ax.kubernetes.kube_object Thread-46: deployment "axops-deployment" created
2017-10-31T20:34:47 INFO ax.kubernetes.kube_object Thread-48: service "axmon" created
2017-10-31T20:34:47 INFO ax.kubernetes.kube_object Thread-48: deployment "axmon-deployment" created
2017-10-31T20:34:47 INFO ax.kubernetes.kube_object Thread-47: service "gateway" created
2017-10-31T20:34:47 INFO ax.kubernetes.kube_object Thread-47: deployment "gateway-deployment" created
2017-10-31T20:35:08 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object gateway-deployment:pods has reached expected status
2017-10-31T20:35:08 INFO ax.platform.platform Thread-47: Successfully created gateway-deployment with code WARN.
2017-10-31T20:35:08 INFO ax.platform.platform Thread-47: Successfully created object gateway-svc.
2017-10-31T20:35:37 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axmon-deployment:pods has reached expected status
2017-10-31T20:35:37 INFO ax.platform.platform Thread-48: Successfully created axmon-deployment with code WARN.
2017-10-31T20:35:37 INFO ax.platform.platform Thread-48: Successfully created object axmon-svc.
2017-10-31T20:35:40 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axops-deployment:pods has reached expected status
2017-10-31T20:35:40 INFO ax.platform.platform Thread-46: Successfully created axops-deployment with code WARN.
2017-10-31T20:35:40 INFO ax.platform.platform Thread-46: Successfully created object axops.
2017-10-31T20:35:40 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
axops                     |  axops-deployment:WARN                                                                                          |  56.83
gateway-svc               |  gateway-deployment:WARN                                                                                        |  23.94
axmon-svc                 |  axmon-deployment:WARN                                                                                          |  53.99

======= WARNING EVENTS =======
axops: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axops-deployment-1534201101-5d47w'}]

gateway-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'gateway-deployment-260125273-0k94w'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-97cfa908-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'gateway-deployment-260125273-0k94w'}]

axmon-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "docker-socket-dir" ', 'name': u'axmon-deployment-2025681693-wf5f1'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axmon-deployment-2025681693-wf5f1'}]

==============================

2017-10-31T20:35:40 INFO ax.platform.platform MainThread: Create step: Backend Micro Services
2017-10-31T20:35:40 INFO ax.platform.platform MainThread: Creating platform objects


NAME                      |  MANIFEST                                                                                                       |  NAMESPACE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
default-http-backend      |  /ax/config/service/argo-all/default-http-backend.yml.in                                                        |  axsys
ingress-controller        |  /ax/config/service/argo-all/ingress-controller.yml.in                                                          |  axsys
axstats                   |  /ax/config/service/argo-all/axstats.yml.in                                                                     |  axsys
applet                    |  /ax/config/service/argo-all/applet.yml.in                                                                      |  axsys
fixturemanager-svc        |  /ax/config/service/argo-all/fixturemanager-svc.yml.in                                                          |  axsys
axamm-svc                 |  /ax/config/service/argo-all/axamm-svc.yml.in                                                                   |  axsys
axopsbootstrap            |  /ax/config/service/argo-all/axopsbootstrap.yml.in                                                              |  axsys
ingress-controller-int    |  /ax/config/service/argo-all/ingress-controller-int.yml.in                                                      |  axsys
redis-svc                 |  /ax/config/service/argo-all/redis-svc.yml.in                                                                   |  axsys
axworkflowadc-svc         |  /ax/config/service/argo-all/axworkflowadc-svc.yml.in                                                           |  axsys
axscheduler-svc           |  /ax/config/service/argo-all/axscheduler-svc.yml.in                                                             |  axsys
axartifactmanager-svc     |  /ax/config/service/argo-all/axartifactmanager-svc.yml.in                                                       |  axsys

2017-10-31T20:35:40 INFO ax.platform.platform Thread-58: Creating applet in namespace axsys ...
2017-10-31T20:35:40 INFO ax.platform.platform Thread-66: Creating axartifactmanager-svc in namespace axsys ...
2017-10-31T20:35:41 INFO ax.platform.platform Thread-57: Creating axstats in namespace axsys ...
2017-10-31T20:35:41 INFO ax.platform.platform Thread-59: Creating fixturemanager-svc in namespace axsys ...
2017-10-31T20:35:41 INFO ax.platform.platform Thread-63: Creating redis-svc in namespace axsys ...
2017-10-31T20:35:42 INFO ax.platform.platform Thread-55: Creating default-http-backend in namespace axsys ...
2017-10-31T20:35:42 INFO ax.platform.platform Thread-56: Creating ingress-controller in namespace axsys ...
2017-10-31T20:35:42 INFO ax.kubernetes.kube_object Thread-58: daemonset "applet" created
2017-10-31T20:35:43 INFO ax.platform.platform Thread-64: Creating axworkflowadc-svc in namespace axsys ...
2017-10-31T20:35:44 INFO ax.platform.platform Thread-61: Creating axopsbootstrap in namespace axsys ...
2017-10-31T20:35:44 INFO ax.kubernetes.kube_object Thread-66: service "axartifactmanager" created
2017-10-31T20:35:44 INFO ax.kubernetes.kube_object Thread-66: deployment "axartifactmanager-deployment" created
2017-10-31T20:35:44 INFO ax.kubernetes.kube_object Thread-57: deployment "axstats-deployment" created
2017-10-31T20:35:45 INFO ax.platform.platform Thread-60: Creating axamm-svc in namespace axsys ...
2017-10-31T20:35:45 INFO ax.platform.platform Thread-65: Creating axscheduler-svc in namespace axsys ...
2017-10-31T20:35:45 INFO ax.platform.platform Thread-62: Creating ingress-controller-int in namespace axsys ...
2017-10-31T20:35:47 INFO ax.kubernetes.kube_object Thread-59: service "fixturemanager" created
2017-10-31T20:35:47 INFO ax.kubernetes.kube_object Thread-59: deployment "fixturemanager-deployment" created
2017-10-31T20:35:47 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axstats-deployment:pods has reached expected status
2017-10-31T20:35:47 INFO ax.platform.platform Thread-57: Successfully created axstats-deployment with code WARN.
2017-10-31T20:35:47 INFO ax.kubernetes.kube_object Thread-63: service "redis" created
2017-10-31T20:35:47 INFO ax.kubernetes.kube_object Thread-63: deployment "redis-deployment" created
2017-10-31T20:35:49 INFO ax.platform.platform Thread-57: Successfully created object axstats.
2017-10-31T20:35:49 INFO ax.kubernetes.kube_object Thread-55: service "default-http-backend" created
2017-10-31T20:35:49 INFO ax.kubernetes.kube_object Thread-55: deployment "default-http-backend" created
2017-10-31T20:35:49 INFO ax.kubernetes.kube_object Thread-61: pod "axops-bootstrap-pod" created
2017-10-31T20:35:50 INFO ax.kubernetes.kube_object Thread-56: configmap "ingress-controller-config" created
2017-10-31T20:35:50 INFO ax.kubernetes.kube_object Thread-56: deployment "ingress-controller-deployment" created
2017-10-31T20:35:50 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object fixturemanager-deployment:pods has reached expected status
2017-10-31T20:35:50 INFO ax.platform.platform Thread-59: Successfully created fixturemanager-deployment with code WARN.
2017-10-31T20:35:51 INFO ax.platform.platform Thread-59: Successfully created object fixturemanager-svc.
2017-10-31T20:35:51 INFO ax.kubernetes.kube_object Thread-64: service "axworkflowadc" created
2017-10-31T20:35:51 INFO ax.kubernetes.kube_object Thread-64: deployment "axworkflowadc-deployment" created
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-60: service "axamm" created
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-60: deployment "axamm-deployment" created
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-65: service "axscheduler" created
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-65: deployment "axscheduler-deployment" created
2017-10-31T20:35:52 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object default-http-backend:pods has reached expected status
2017-10-31T20:35:52 INFO ax.platform.platform Thread-55: Successfully created default-http-backend with code WARN.
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-62: configmap "ingress-controller-int-config" created
2017-10-31T20:35:52 INFO ax.kubernetes.kube_object Thread-62: deployment "ingress-controller-int-deployment" created
2017-10-31T20:35:52 INFO ax.platform.platform Thread-55: Successfully created object default-http-backend.
2017-10-31T20:35:53 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axops-bootstrap-pod:pods has reached expected status
2017-10-31T20:35:53 INFO ax.platform.platform Thread-61: Successfully created axops-bootstrap-pod with code WARN.
2017-10-31T20:35:53 INFO ax.platform.platform Thread-61: Successfully created object axopsbootstrap.
2017-10-31T20:35:53 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object ingress-controller-deployment:pods has reached expected status
2017-10-31T20:35:53 INFO ax.platform.platform Thread-56: Successfully created ingress-controller-deployment with code WARN.
2017-10-31T20:35:53 INFO ax.platform.platform Thread-56: Successfully created object ingress-controller.
2017-10-31T20:35:54 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axamm-deployment:pods has reached expected status
2017-10-31T20:35:54 INFO ax.platform.platform Thread-60: Successfully created axamm-deployment with code WARN.
2017-10-31T20:35:54 INFO ax.platform.platform Thread-60: Successfully created object axamm-svc.
2017-10-31T20:35:55 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object ingress-controller-int-deployment:pods has reached expected status
2017-10-31T20:35:55 INFO ax.platform.platform Thread-62: Successfully created ingress-controller-int-deployment with code WARN.
2017-10-31T20:35:55 INFO ax.platform.platform Thread-62: Successfully created object ingress-controller-int.
2017-10-31T20:36:09 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axworkflowadc-deployment:pods has reached expected status
2017-10-31T20:36:09 INFO ax.platform.platform Thread-64: Successfully created axworkflowadc-deployment with code WARN.
2017-10-31T20:36:10 INFO ax.platform.platform Thread-64: Successfully created object axworkflowadc-svc.
2017-10-31T20:36:11 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object redis-deployment:pods has reached expected status
2017-10-31T20:36:11 INFO ax.platform.platform Thread-63: Successfully created redis-deployment with code WARN.
2017-10-31T20:36:11 INFO ax.platform.platform Thread-63: Successfully created object redis-svc.
2017-10-31T20:40:22 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axartifactmanager-deployment:pods has reached expected status
2017-10-31T20:40:22 INFO ax.platform.platform Thread-66: Successfully created axartifactmanager-deployment with code WARN.
2017-10-31T20:40:23 INFO ax.platform.platform Thread-66: Successfully created object axartifactmanager-svc.
2017-10-31T20:40:24 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object axscheduler-deployment:pods has reached expected status
2017-10-31T20:40:24 INFO ax.platform.platform Thread-65: Successfully created axscheduler-deployment with code WARN.
2017-10-31T20:40:24 INFO ax.platform.platform Thread-65: Successfully created object axscheduler-svc.
2017-10-31T20:40:27 INFO ax.platform.ax_monitor kube-pods-monitor: Thread kube-pods-monitor: object applet:pods has reached expected status
2017-10-31T20:40:27 INFO ax.platform.platform Thread-58: Successfully created applet with code WARN.
2017-10-31T20:40:27 INFO ax.platform.platform Thread-58: Polling till healthy to make sure rest of components of applet are up and running ...
2017-10-31T20:40:31 INFO ax.platform.platform Thread-58: Successfully created applet.
2017-10-31T20:40:31 INFO ax.platform.platform MainThread:

Platform Create SUCCESSFULLY. Report:

NAME                      |  STATUS                                                                                                         |  TIME (sec)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
ingress-controller        |  ingress-controller-deploy:WARN                                                                                 |  11.19
axamm-svc                 |  axamm-deployment:WARN                                                                                          |  9.22
default-http-backend      |  default-http-backend:WARN                                                                                      |  10.18
redis-svc                 |  redis-deployment:WARN                                                                                          |  30.3
axstats                   |  axstats-deployment:WARN                                                                                        |  7.55
axopsbootstrap            |  axops-bootstrap-pod:WARN                                                                                       |  8.7
axworkflowadc-svc         |  axworkflowadc-deployment:WARN                                                                                  |  26.69
axartifactmanager-svc     |  axartifactmanager-deploym:WARN                                                                                 |  282.81
fixturemanager-svc        |  fixturemanager-deployment:WARN                                                                                 |  9.78
ingress-controller-int    |  ingress-controller-int-de:WARN                                                                                 |  10.22
applet                    |  applet:WARN / applet:OK                                                                                        |  291.16
axscheduler-svc           |  axscheduler-deployment:WARN                                                                                    |  279.01

======= WARNING EVENTS =======
ingress-controller: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'ingress-controller-deployment-461330486-w0wvd'}]

axamm-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axamm-deployment-3760474450-kj8bx'}]

default-http-backend: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'default-http-backend-2782281302-5tqvh'}]

redis-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'redis-deployment-4047683866-330hk'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "pvc-9945bf6f-be7a-11e7-ab7a-0af8e172eeda" ', 'name': u'redis-deployment-4047683866-330hk'}]

axstats: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axstats-deployment-4019305794-q21p9'}]

axopsbootstrap: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axops-bootstrap-pod'}]

axworkflowadc-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axworkflowadc-deployment-388509999-vmh0v'}]

axartifactmanager-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axartifactmanager-deployment-2059223331-k6kc9'}]

fixturemanager-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'fixturemanager-deployment-456989105-ngcxv'}]

ingress-controller-int: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'ingress-controller-int-deployment-2045068612-7rt98'}]

applet: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "handshake-socket" ', 'name': u'applet-6h97j'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "database" ', 'name': u'applet-6h97j'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "container-logs" ', 'name': u'applet-6h97j'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'applet-6h97j'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "container-logs" ', 'name': u'applet-tgsr3'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "database" ', 'name': u'applet-tgsr3'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "handshake-socket" ', 'name': u'applet-tgsr3'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'applet-tgsr3'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "database" ', 'name': u'applet-zxrr6'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "handshake-socket" ', 'name': u'applet-zxrr6'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "database" ', 'name': u'applet-kx71h'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "container-logs" ', 'name': u'applet-zxrr6'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'applet-zxrr6'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "handshake-socket" ', 'name': u'applet-kx71h'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "container-logs" ', 'name': u'applet-kx71h'}, {'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'applet-kx71h'}]

axscheduler-svc: [{'reason': u'SuccessfulMountVolume', 'message': u'MountVolume.SetUp succeeded for volume "default-token-vlfnw" ', 'name': u'axscheduler-deployment-3302683748-9x3cm'}]

==============================

2017-10-31T20:40:31 INFO ax.platform.platform MainThread: Updating application managers
2017-10-31T20:40:31 INFO ax.platform.platform MainThread: --- updating claudia
2017-10-31T20:40:32 ERROR ax.cluster_management.app.cluster_installer MainThread: Cluster name id is None!
Traceback (most recent call last):
  File "/ax/python/ax/cluster_management/app/cluster_installer.py", line 271, in install_and_run_platform
    platform.start()
  File "/ax/python/ax/platform/platform.py", line 336, in start
    a = Application(app, client=self.kubectl)
  File "/ax/python/ax/platform/application.py", line 62, in __init__
    assert cluster_name_id, "Cluster name id is None!"
AssertionError: Cluster name id is None!
2017-10-31T20:40:32 ERROR ax.cluster_management.argo_cluster_manager MainThread: Cluster name id is None!
Please manually check the cluster status and retry installation with same command if the error is transient.
Traceback (most recent call last):
  File "/ax/python/ax/cluster_management/argo_cluster_manager.py", line 86, in parse_args_and_run
    getattr(self, cmd)(args)
  File "/ax/python/ax/cluster_management/argo_cluster_manager.py", line 267, in install_argo_only
    PlatformOnlyInstaller(platform_install_config).run()
  File "/ax/python/ax/cluster_management/app/cluster_installer.py", line 527, in run
    cluster_dns, username, password = self._ci_installer.install_and_run_platform()
  File "/ax/python/ax/cluster_management/app/cluster_installer.py", line 279, in install_and_run_platform
    raise RuntimeError(install_platform_failure_message)
RuntimeError: Cluster name id is None!
Please manually check the cluster status and retry installation with same command if the error is transient.

 !!! Operation failed due to runtime error: Cluster name id is None!
Please manually check the cluster status and retry installation with same command if the error is transient.
```

--- a/platform/source/lib/ax/platform/platform.py
+++ b/platform/source/lib/ax/platform/platform.py
@@ -305,6 +305,7 @@ class AXPlatform(object):
         else:
             self._replacing = self._generate_replacing_for_user_provisioned_cluster()
 
+        os.environ["AX_CLUSTER_NAME_ID"] = self._cluster_name_id
         logger.debug("Replacing ENVs: %s", self._replacing)
 
         # TODO: remove component's dependencies to AXOPS_EXT_DNS env (#32)

